toolkit/tools/internal/pkggraph/pkggraph_test.go

```diff
@@ -615,35 +644,37 @@ func TestConditionalLookupBasic(t *testing.T) {
 func TestConditionalLookupMulti(t *testing.T) {
 	g := NewPkgGraph()
 	err := addNodesHelper(g, unresolvedNodes)
-	assert.NoError(t, err)
-	assert.NotNil(t, g)
+	require.NoError(t, err)
+	require.NotNil(t, g)
 
-	// Should get D2 (<= 2)
+	// Should get nothing, closest is D2 (<= 2), but we have no garantuee that it will be >=1 when resolved
```

Comment:
I realized I'm not sure what a package with version `<=` means. Is it the "provides" from this package? Or maybe it's just invalid to have a package with a version anything else than `=`? If it's a "provides", then `<= 2` would mean that it guarantees to provide version 2 of the package, so this should be satisfactory for a `>= 1` requirement.